src/auth.ts

```diff
@@ -0,0 +1,69 @@
+import Ain from '@ainblockchain/ain-js';
+import axios from 'axios';
+import stringify = require('fast-json-stable-stringify');
+import { User } from './types';
+
+export default class Auth {
+  private baseUrl: string;
+  public ain: Ain;
+
+  constructor(baseUrl: string, ain: Ain) {
+    this.baseUrl = `${baseUrl}/auth`;
+    this.ain = ain;
+  }
+
+  setBaseUrl(baseUrl: string) {
+    this.baseUrl = `${baseUrl}/auth`;
+  }
+
+  getUser(appId: string, userId: string): Promise<User> {
+    const data = {
+      appId,
+      userId,
+      timestamp: Date.now(),
+    };
+    const signature = this.ain.wallet.sign(stringify(data));
+    return axios
+      .get(`${this.baseUrl}/user`, {
+        data: { data, signature },
+      })
+      .then((res) => res.data)
```

Comment:
How about res.data.data?  
res.data like this 
```
{ code, message, data }
```

Reply:
right! fixed!